### ISSUE TITLE
Hackathon?

### ISSUE THREAD

**Issue by danr** [ollef/sixten#63]
Hej!

Will there be a sixten hackathon soon? I could come visit in Oslo.

Cheers,
Dan

**Comment by ollef** [comment]
Hej Dan! Kul initiativ!

I'd love that!

It seems like I'm not free a full weekend until January, so if you'd like to stay for a few days perhaps that's best. When would you like to come?

**Comment by danr** [comment]
Januari sounds great! Maybe the weekend 13-14?

**Comment by ollef** [comment]
Yeah, January 13-14 works for me. You can sleep on my couch if you want. :) 

@ehamberg, others: you're welcome to join us!

**Comment by ehamberg** [comment]
Would love to take part in a Sixten hackathon at some point, and I'd be happy to host it in the new Blank office in the city centre. However, I'll be travelling all of January, so my egotistical vote is for either postponing it to after 5 February – or doing another one. ;)

**Comment by ollef** [comment]
Ah, bummer. I wouldn't mind having multiple hackathons, but maybe it'd be more fun to try to do one all together if we can. If we are to postpone to after February 5 it likely can't be until March due to my plans.

Any preferences @danr? Should we still go for the original date above? Otherwise I can set up a doodle and we can see if we can find a date that works for everyone.

**Comment by danr** [comment]
I would be happy to come in january and also to crash on your couch :)

I would also be happy to meet you @ehamberg at some point!

**Comment by ollef** [comment]
Cool, let's do it! :)

**Comment by ollef** [comment]
Hey Dan,

Sorry to yoyo on this, but something's come up: a close friend's getting married (on a somewhat short notice) and I feel like I have to go. Can we do it the week after, January 20-21?

**Comment by danr** [comment]
Hey nemas problemas... That weekend is worse for me, how does January 27-28 look for you? :)

**Comment by ollef** [comment]
That works! :)

**Comment by danr** [comment]
Ok! Let's do it then!

**Comment by danr** [comment]
It happened!

Here's the video: https://asciinema.org/a/159730

**Comment by ehamberg** [comment]
We should do another one! I don't have any travel plans in the near future, so I can hopefully attend the next one. :)

**Comment by ollef** [comment]
I'm extremely down! Anyone else? Can we lure @danr here again?
